sdk/python/tests/test_kfp_samples_report.txt

```diff
@@ -2,15 +2,15 @@ SUCCESS: add_pod_env.py
 SUCCESS: artifact_location.py
 SUCCESS: basic.py
 FAILURE: basic_no_decorator.py
-SUCCESS: coin.py
+FAILURE: coin.py
```

Comment:
So this one is reverting back to failure @drewbutlerbb4 ?

Reply:
coin.py uses parameter passing from task outputs to conditions which is [not yet supported in tekton](https://github.com/tektoncd/pipeline/issues/2336). Right now I have the compiler exit gracefully when it sees this so as to not cause confusion.

Reply:
This is the same reason recursive_do_while.py fails as well

Reply:
Ok - how come it was set to 'SUCCESS' before?

Reply:
There's a bug in Tekton where we cannot pass output parameters for condition. We get someone in the community to look in this issue https://github.com/tektoncd/pipeline/issues/2336. Once the bug is fixed, then we can implement with output parameters with Tekton 0.12 since Andrew already have that output parameters code for compiler in a different branch.

Reply:
The compiler only failed if we raise an error. Otherwise it will be ignored and passed.

Reply:
It passed before because the compiler ignored conditions altogether. The pipeline would compile fine, but the conditions would not show up in the resulting tekton pipeline yaml.